setup.py

```diff
@@ -35,26 +30,39 @@ def get_install_requires():
 install_requires, dependency_links = get_install_requires()
 
 
+def get_long_description():
+    with open('README.rst', 'r') as f:
+        return f.read()
```

Comment:
I could argue the same. I don't really see any benefit of replacing `read()` with a generator for the functions which read the version and requirements.txt. IMO there's no need to change how you read text files sized a few lines each.